config.toml

```diff
@@ -140,6 +140,10 @@ account_prefix = 'cosmos'
 #   https://hermes.informal.systems/commands/keys/index.html#adding-keys
 key_name = 'testkey'
 
+# Specify the folder used to store the keys. Optional
+# If this is not specified then the hermes home folder is used.
+# key_store_folder = '/home/.hermes_keys'
```

Comment:
Let's also use the same default value we have currently, ie. `$HOME/.hermes/keys`